templates/app/orchestrator.yaml

```diff
@@ -71,6 +88,8 @@ objects:
       triggers:
       - type: ConfigChange
 parameters:
+- name: DATABASE_PORT
+  value: '5432'
```

Comment:
Should this be a number? or does it not matter?

Reply:
Nope, I think it actually fails with a number since it's going to be an env var.